### PR TITLE
Dark mode overrides for material components: checkbox, drowdown, textfield, textarea and some table styles.

### PR DESCRIPTION
**Pull request by isoos**
#4416

### CODE REVIEW

File: app/lib/service/entrypoint/frontend.dart [hunk 1]
```diff
@@ -63,7 +63,7 @@ Future<void> watchForResourceChanges() async {
   void setupWatcher(
       String name, String path, FutureOr<void> Function() updateFn) {
     final w = Watcher(path, pollingDelay: Duration(seconds: 3));
-    final subs = w.events.debounce(Duration(milliseconds: 200)).listen(
+    final subs = w.events.debounce(Duration(milliseconds: 2500)).listen(
```
Comment (sigurdm) [inline app/lib/service/entrypoint/frontend.dart:66]: Is this related?

**Review by sigurdm — APPROVED** [review]

File: app/lib/service/entrypoint/frontend.dart (hunk 1 above)
Comment (isoos) [reply app/lib/service/entrypoint/frontend.dart:66]: Yes, although only in development mode with `--watch` flag: without it on my machine the rebuild got triggered way too fast, often failing with syntax error due to unfinished line being saved by the IDE. A slightly longer time provides much better workflow.
